packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -102,8 +103,8 @@ const createProviderInstance = (
             return new GoogleProvider(tokens, environment);
         case 'fileSystem':
             return new FileSystemProvider();
-        default:
```

Comment:
This is an example of having default behavior (in this case throwing).

With strict mapping it is not possible to not handle all cases (all providers)